src/vanilla/utils/atomWithRefresh.ts

```diff
@@ -35,10 +35,10 @@ export function atomWithRefresh<Value, Args extends unknown[], Result>(
       return read(get, options as never)
     },
     (get, set, ...args: Args) => {
-      if (args.length === 0) {
-        set(refreshAtom, (c) => c + 1)
```

Comment:
I think the contract is, if no args, refresh. So, I would like to keep this as is.